src/interactive/Tooltip.jsx

```diff
@@ -178,7 +184,7 @@ class Tooltip extends React.PureComponent {
 											reset
 										/>
 									)}
-									{content}
+									{content || children}
```

Comment:
nitpick should this be reversed to make it clear that the deprecated prop is the fallback? `children || content`